modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/ShortcutManager.js

```diff
@@ -42,6 +42,9 @@ const ctrlOrMeta = (event) =>
 const isEditableField = (element) =>
 	!!element.closest('.page-editor__editable');
 
+const isEditingEditableField = () =>
+	!!document.activeElement.getAttribute('contenteditable');
```

Comment:
I'm unsure as to whether or not this is the best way to determine if a user is actively editing text. Is there a better approach?

Reply:
I'm not sure either, as FEI and JS aren't strong suites for me.  Assuming your changes work, everything _looks_ in order, so I'll forward to echo for their review.